pkg/cmd/backuprestore/backupserver.go

```diff
@@ -0,0 +1,127 @@
+package backuprestore
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+
+	"k8s.io/klog/v2"
+
+	"github.com/adhocore/gronx/pkg/tasker"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var shutdownSignals = []os.Signal{os.Interrupt, syscall.SIGTERM}
+
+type backupServer struct {
+	schedule  string
+	timeZone  string
+	scheduler *tasker.Tasker
+	backupOptions
+}
+
+func NewBackupNoConfigCommand(ctx context.Context, errOut io.Writer) *cobra.Command {
```

Comment:
```suggestion
func NewBackupServer(ctx context.Context) *cobra.Command {
```

let's name it properly already, why do you need to add the stderr if you're just using `os.Stderr` anyway?

Reply:
done 👍🏽